libass/ass_shaper.c

```diff
@@ -758,7 +758,7 @@ static void shape_fribidi(ASS_Shaper *shaper, GlyphInfo *glyphs, size_t len)
  * \param shaper shaper instance
  * \param kern toggle kerning
  */
-void ass_shaper_set_kerning(ASS_Shaper *shaper, int kern)
+void ass_shaper_set_kerning(ASS_Shaper *shaper, bool kern)
 {
 #ifdef CONFIG_HARFBUZZ
     shaper->features[KERN].value = !!kern;
```

Comment:
I usually use `!!` for `int` to `bool` conversion, but here it's in reverse. Maybe it's better to write directly as `kern ? 1 : 0`?

Reply:
The `!!` isn’t needed at all, as `_Bool` is an integer that can only be 0 or 1. I’ve now removed it.